ui/app/js/controllers/rules_controller.js

```diff
@@ -170,12 +170,23 @@ function($scope, $routeParams, $location, $timeout, Rules, Search, $modal, $rout
     return !!(false || $scope.filters.search.length);
   };
 
+  $scope.returnRuleType = function() {
+    var firstRuleId = $scope.rules[0].rule_id;
+    return $scope.rules.every(function(rule) {
+      return rule.rule_id === firstRuleId;
+    }) ? "Revision" : "All Rules";
+  };
+
   $scope.orderRules = function(rule) {
     // Rules are sorted by priority. Rules that are pending (ie: still just a Scheduled Change)
```

Comment:
Please keep this comment just above in the "else" block -- it doesn't relate to the new code you've added.